src/android/TPDate.java

```diff
@@ -57,7 +57,7 @@ private void reinit() {
           // See full example at https://github.com/instacart/truetime-android/blob/master/app/src/main/java/com/instacart/library/sample/App.java
           TrueTime.build()
             .withNtpHost(DEFAULT_NTP_HOST)
-            .withSharedPreferences(cordova.getActivity().getApplication()) //3.2
+            // .withSharedPreferences(cordova.getActivity().getApplication()) //3.2
```

Comment:
any reason to keep the comments?

Reply:
Not in particular; though, if TrueTime ends up fixing the flaws in the disk caching system, the code required to start using it again is there.